scripts/delete-storage.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+
+# shellcheck disable=SC1091 # Not following.
+source "$SCRIPT_DIR"/init-env.sh
+
+mkdir -p ./temp
+
+# shellcheck disable=SC2002 # Useless cat.
+cat ./test-target/storage.yaml | "$SCRIPT_DIR"/mo >./temp/rendered-test-storage.yaml
```

Comment:
Does `"$SCRIPT_DIR"/mo ./test-target/storage.yaml` work?

Reply:
Yes, it does:
```
+ cat ./test-target/storage.yaml
+ ././scripts/mo
+ oc delete --filename ./temp/rendered-test-storage.yaml -n tnf --ignore-not-found=true
warning: deleting cluster-scoped resources, not scoped to the provided namespace
persistentvolume "test-pv" deleted
persistentvolumeclaim "test-pvc" deleted
+ rm ./temp/rendered-test-storage.yaml
```
It just issues a warning because the PV is not namespaced while the PVC is. But that expected.

Reply:
Using `cat` in this case is incorrect. Please remove comment `shellcheck disable` as well.

Reply:
This pattern for rendering the manifests was already used for several resources. If you think it's not correct I believe it's better to do a refactor in a follow-up PR.